systemjs/react-hello-world/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "react-hello-world",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "babel src --out-dir dist --source-maps"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/cli": "^7.6.0",
+    "@babel/core": "^7.6.0",
```

Comment:
Does Babel core still need to be a dependency when using the CLI?

Reply:
Yes

<img width="752" alt="image" src="https://user-images.githubusercontent.com/5524384/65532461-14c0b380-deb9-11e9-842d-60ff9954c86f.png">